src/Template/Element/sentences/index/all.ctp

```diff
@@ -0,0 +1,32 @@
+<?php
+  $baseUrl = $this->Url->build(['action' => 'show_all_in']);
+?>
+
+<?php foreach ($stats as $milestone => $languages): ?>
+  <h3 class="header-with-hline">
+    <span class="hline">
+      <span class="text">
+        <?= h(format(
+                __n("{n}+ sentence", "{n}+ sentences", $milestone),
+                ['n' => $this->Number->format($milestone)]
+            )) ?>
+      </span>
+    </span>
+  </h3>
+  <div class="language-list">
+    <ul>
+      <?php
+        $l = h(json_encode($languages));
+      ?>
+
+      <li ng-repeat="lang in <?= $l ?>">
+        <a ng-href="<?= $baseUrl ?>/{{lang.code}}/none">
+          <img class="language-icon" width="30" height="20"
+               ng-attr-title="{{lang.name ? lang.name : lang.code}}"
```

Comment:
Is it really necessary to show the tooltip on the flag when the language name is displayed right next to it? (This is more a general question which also concerns other pages outside the scope of this PR.)
Mobile users won't see it anyways.